CMakeLists.txt

```diff
@@ -656,7 +698,10 @@ install(DIRECTORY extlibs/ttmath/ttmath
 # Prepare pkg-config file
 
 set(NETWORKIT_LIBNAME networkit)
-execute_process(COMMAND python3 -c "from version import version; print(version, end='')"
+if(${Python3_Interpreter_FOUND})
+	set(NETWORKIT_CMAKE_PYTHON_INTERPRETER ${Python3_EXECUTABLE})
+endif()
```

Comment:
What if the interpreter is not found? Can we add something like a `REQUIRED` clause to `find_package`?

Reply:
Sure, should have used `REQUIRED` in the first place.

Reply:
But then we can also remove this `if`.